include/tier_vesting.hpp

```diff
@@ -8,54 +12,52 @@ class [[eosio::contract]] tier_vesting : public contract {
   using contract::contract;
 
   [[eosio::action]]
-  void addtier(name tier_id, asset total_amount, time_point_sec start_date, std::string name);
+  void addtier(name tier_id, asset total_amount, std::string name);
 
   [[eosio::action]]
-  void release(name tier_id, double percent);
+  void removetier(name tier_id);
 
   [[eosio::action]]
-  void addlock(name sender, name owner, name tier_id, asset amount);
+  void release(name tier_id, asset amount);
 
   [[eosio::action]]
   void claim(name owner, uint64_t lock_id);
 
   [[eosio::action]]
-  void addtoken(name token_contract, asset token);
+  void addlock(name sender, name owner, name tier_id, asset amount, std::string note);
 
   [[eosio::action]]
   void removelock(uint64_t lock_id);
-
```

Comment:
I removed ability to handle different tokens for now, it's too complicated and we don't need it initially.